mne/decoding/time_gen.py

```diff
@@ -1049,11 +1136,13 @@ def plot_times(self, train_time, title=None, xmin=None, xmax=None,
         fig : instance of matplotlib.figure.Figure
             The figure.
         """
-        if (not isinstance(train_time, float) and
-            not (isinstance(train_time, (list, np.ndarray)) and
-                 np.all([isinstance(time, float) for time in train_time]))):
-            raise ValueError('train_time must be float | list or array of '
-                             'floats. Got %s.' % type(train_time))
+        not_float = not isinstance(train_time, float)
+        if not_float:
+            # Check that train_time is an array of floats
+            if not (isinstance(train_time, (list, np.ndarray)) and
+                    np.all([isinstance(ii, float) for ii in train_time])):
```

Comment:
silly question. Why not simply convert the list to an array and check if the array is `float64`?